nvidia-driver-installer/cos/daemonset-nvidia-mig.yaml

```diff
@@ -101,7 +101,7 @@ spec:
           mountPath: /root
         - name: cos-tools
           mountPath: /build/cos-tools
-      - image: "gcr.io/gke-release/nvidia-partition-gpu@sha256:e226275da6c45816959fe43cde907ee9a85c6a2aa8a429418a4cadef8ecdb86a"
+      - image: "gcr.io/gke-release/nvidia-partition-gpu@sha256:116be6b7335c1d34366223b9a3780fe80d862fcf06cd2c580426fdc1697af693"
```

Comment:
I didn't find this image on gke-release. Could you double check the image hash?

Reply:
Updated to use image from gke-release instead of gke-release-staging